lib/airbrake-ruby/notifier.rb

```diff
@@ -42,6 +44,12 @@ def initialize(user_config)
     ##
     # @macro see_public_api_method
     def notify(exception, params = {})
+      if !@workers_spawned || @pid != Process.pid
```

Comment:
Do you really need `!@workers_spawned` check? I think `@pid != Process.pid` should be enough...